Uploaded image scale to screen, zoom buttons

sebmid opened the pull request:
# Changes


ZoomButtons,tsx **New**
+ New buttons for controlling zoom
+ also reset zoom button

moveImage.tsx
+ Now uses a scale factor to scale the image based on the size of the image
+ then does transform to center the image after the scaling has been done

splitview.tsx
+ just adds zoombuttons to component

sebmid committed:
merge dev

sebmid committed:
Move image sets the zoomlevel based on a scalefactor which is based on the width/height of the image.
Then sets an initial translate which centers the image based on the imagewidth and scale

sebmid committed:
Refactor SplitView component to adjust marker positions based on scaleFactor

sebmid committed:
Add ZoomButtons component to UI folder for controlling the zoom without needing to use the mousewheel or trackpad zoom

sebmid committed:
Add comments and refactor stuff

sebmid committed:
Add ZoomButtons component to SplitView for controlling the zoom level

sebmid committed:
fix light mode for buttons

sebmid committed:
Merge branch 'dev' into f_sm_fitImage

sebmid committed:
Fix import statement in moveImage.tsx

sebmid committed:
Merge branch 'f_sm_fitImage' of https://github.com/TheAtlasRepository/ImgPDF2Map into f_sm_fitImage

ostepizza commented on frontend/components/ui/ZoomButtons.tsx:
Preferably use Button-component 😿

sebmid replied:
oki 🥉

ostepizza reviewed (changes requested):
pls look comment sorry 😭

sebmid committed:
Refactor ZoomButtons component to use button component

ostepizza reviewed (approved):
"real man doing a pog-face"
![image](https://github.com/TheAtlasRepository/ImgPDF2Map/assets/32636970/78ba153a-0445-4cdc-9c3f-10c68e66325b)

sebmid committed:
Update sniperScope component to add cursor-pointer to confirm and cancel buttons

sebmid committed:
Add ResetIcon to ZoomButtons component

sebmid committed:
Added tooltops to buttons on hover

ostepizza committed:
Minor style changes

Moved tooltip to the left to avoid overlapping buttons
Somewhat improved contrast on light mode
Replaced + and - with radix icons

sebmid committed:
Update button styles for zoom variant

sebmid committed:
Update button styles for zoom variant

Trivinyx reviewed (approved):
Goood
